.gitattributes

```diff
@@ -1,4 +1,3 @@
 sql/parser/sql.go -diff
 ui/embedded.go -diff
-*.pb.go -diff
-*.pb.gw.go -diff
+*.pb.* -diff
```

Comment:
I think @dt mentioned that an upshot to `binary` is that `git` won't attempt to merge the files. Haven't looked further, but maybe going back to `binary` is the way to go for all of these.